components/organisms/OverviewSidebar.tsx

```diff
@@ -7,6 +7,23 @@ import { useRef, useState } from "react";
 import { SupportDisplay } from "../molecules/SupportDisplay";
 import { NodeStatusUpdate } from "../atoms/NodeStatusUpdate";
 
+function isVersionLessThan(version1?: string, version2?: string): boolean {
```

Comment:
should we implement this something like compare functions?
x < y -> -1
x == y -> 0
x > y -> 1

This way sequence of parameters won't matter; And this is like standard implementations while comparing objects.

Also, please add comment on function describing its usage.